src/site/content/en/blog/quintoandar/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: How QuintoAndar increased conversion rates and pages per session by improving page performances
+subhead: A project focused on optimizing Core Web Vitals and migrating to Next.js resulted in a 5% increase in conversion rates and a 87% increase in pages per session.
+authors:
+  - jlwagner
+date: 2021-12-06
+hero: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/PHqBoMdg3WgcQWWIIkWj.jpg
+alt: A picture of a city skyline on a clear, sunny day. At left is a blue box with the QuintoAndar logo set within it in white.
+description: |
+  TODO
```

Comment:
```suggestion
  A project focused on optimizing Core Web Vitals and migrating to Next.js resulted in a 5% increase in conversion rates and a 87% increase in pages per session.
```

Reply:
TODO: Review this description. At current, it's just the same as the headline. If that's fine, I can just commit the above suggestion.

cc: @danielasy